app/src/main/java/com/babylon/wallet/android/data/repository/state/StateRepository.kt

```diff
@@ -37,7 +38,9 @@ interface StateRepository {
 
     suspend fun getMoreNFTs(account: Network.Account, resource: Resource.NonFungibleResource): Result<Resource.NonFungibleResource>
```

Comment:
I think this and the `GetMoreNFTsUseCase` I would rename them as "getNextPageOfNFTs" because these names looks like "get free nfts".
Or this is only me thinking this way 😄

Reply:
Done